src/hx711-element.ts

```diff
@@ -0,0 +1,417 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ElementPin, GND, VCC } from './pin';
+
+@customElement('wokwi-hx711')
+export class HX711Element extends LitElement {
+  @property() type: '5kg' | '50kg' | 'gauge' | undefined;
+
+  readonly pinInfo: ElementPin[] = [
+    { name: 'VCC', y: 53, x: 7, number: 1, signals: [VCC()] },
+    { name: 'DT', y: 35, x: 7, number: 2, signals: [] },
+    { name: 'SCK', y: 44, x: 7, number: 3, signals: [] },
+    { name: 'GND', y: 25.5, x: 7, number: 4, signals: [GND()] },
+  ];
+
+  get dimension() {
+    switch (this.type) {
+      case '50kg':
+        return { width: '580', height: '430' };
+      case '5kg':
+        return { width: '507', height: '269' };
+      case 'gauge':
+        return { width: '509', height: '200' };
+      default:
+        return { width: '580', height: '430' };
```

Comment:
why not return width/height as numbers instead of strings? then you can remove the conversion to integer in `render()`